app/javascript/packages/document-capture/services/upload.tsx

```diff
@@ -45,16 +44,13 @@ export function toFormData(object) {
  *
  * @return {UploadFormEntryError}
  */
-export function toFormEntryError(uploadFieldError) {
+export function toFormEntryError(uploadFieldError: UploadFieldError) {
   const { field, message } = uploadFieldError;
   const formEntryError = new UploadFormEntryError(message);
   formEntryError.field = field;
   return formEntryError;
 }
 
-/**
- * @type {import('../context/upload').UploadImplementation}
- */
 async function upload(payload, { method = 'POST', endpoint, csrf }) {
```

Comment:
If we wanted to assign the type during the function creation, we could do something like:

```suggestion
const upload: UploadImplemented = async (payload, { method = 'POST', endpoint, csrf }) => {
```